src/libraries/System.Numerics.Tensors/ref/System.Numerics.Tensors.netcore.cs

```diff
@@ -83,18 +90,25 @@ public partial interface IReadOnlyTensor<TSelf, T> : System.Collections.Generic.
         bool TryCopyTo(scoped System.Numerics.Tensors.TensorSpan<T> destination);
         bool TryFlattenTo(scoped System.Span<T> destination);
     }
-    [System.Diagnostics.CodeAnalysis.Experimental("SYSLIB5001", UrlFormat = "https://aka.ms/dotnet-warnings/{0}")]
-    public partial interface ITensor<TSelf, T> : System.Collections.Generic.IEnumerable<T>, System.Collections.IEnumerable, System.Numerics.Tensors.IReadOnlyTensor<TSelf, T> where TSelf : System.Numerics.Tensors.ITensor<TSelf, T>
+    [System.Diagnostics.CodeAnalysis.ExperimentalAttribute("SYSLIB5001", UrlFormat="https://aka.ms/dotnet-warnings/{0}")]
```

Comment:
Just curious - did you use https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/updating-ref-source.md to generate this ref file automatically?

Reply:
Yep! But then I did still have to fix some stuff manually.